### ISSUE TITLE
[Suggestion] A guidebook about keeping bees

### ISSUE THREAD

**Issue by Quaerere-Git** [GregTech6/gregtech6#92]
It is somewhat confusing for noobs like me when it comes to keeping bees, so it might be a good idea to add a guidebook explaining the mechanism of bees living, producing and breeding.

**Comment by GregoriusT** [comment]
Yeah I probably need one of those for the Bumblebees. They are quite difficult if you dont know what the heck you are doing.

**Comment by tyrael93** [comment]
Yeah, having a book about bumblekeeping would definitely be an improvement over the current "just look around at the changelog and read the names of the breeding pairs backwards" implementation :smile:
